test/cpp/qps/client_async.cc

```diff
@@ -156,7 +156,7 @@ class AsyncClient : public Client {
       std::function<ClientRpcContext*(int, TestService::Stub*,
                                       const SimpleRequest&)> setup_ctx)
       : Client(config),
-        channel_lock_(config.client_channels()),
+        channel_lock_(new std::mutex[config.client_channels()]),
```

Comment:
ditto as for std::lock_guard. Is std::mutex supported by the uberold compilers that didn't have std::vector?